include/intx/int128.hpp

```diff
@@ -239,21 +249,21 @@ constexpr bool operator==(uint128 x, uint128 y) noexcept
     //         much better than __int128 where it uses vector instructions.
     // GCC8: generates a bit worse cmp based code
     //       although it generates the xor based one for __int128.
-    return (x.lo == y.lo) & (x.hi == y.hi);
+    return (x.words[0] == y.words[0]) & (x.words[1] == y.words[1]);
```

Comment:
Why not use `lo_half/hi_half` here?

Reply:
I guess the answer is to prepare for a non-recursive implementation, but then why have those helpers?

Reply:
Originally I was using `hi_half` and `lo_half` then I switched to `.hi` and `.lo`. The helpers are still available because used in single recursive function.